Packs/jamf/Integrations/jamfV2/jamfV2_description.md

```diff
@@ -1,14 +1,15 @@
+This integration supports three authentication methods:
 
-#### Server URL:
-Your server URL (e.g.: https://yourdomain.jamfcloud.com)
-#### Username:
-The username of a user with relevant permissions to your Jamf Pro admin console
-#### Password:
-The login password for the above username.
+1. **Basic authentication:** Deprecated starting from version *10.35.0* of Jamf. For more information, click [here](https://developer.jamf.com/jamf-pro/docs/classic-api-authentication-changes).
```

Comment:
```suggestion
1. **Basic authentication:** Deprecated starting from version *10.35.0* of Jamf. For more information, click [here](https://developer.jamf.com/jamf-pro/docs/classic-api-authentication-changes).
Note: This method is used as fallback if other authentication method failed, which will result in general failure if basic authentication is disabled in JAMF.
```

Reply:
This is not exactly correct, since it will be used as a fallback only for the second method, not "if other authentication method failed", and I alredy wrote that at the bottem.